lib/service/service.go

```diff
@@ -5422,6 +5492,26 @@ func (process *TeleportProcess) initMinimalReverseTunnel(listeners *proxyListene
 	return minimalWebServer, nil
 }
 
+func (process *TeleportProcess) readOrInitPeerStatelessResetKey() (*quic.StatelessResetKey, error) {
```

Comment:
should we use the sqlite to store the key? That would help with future QUIC for agent-agent connecitons

Reply:
see https://github.com/gravitational/teleport/pull/47216#discussion_r1789947444